src/linter/root.go

```diff
@@ -1395,8 +1395,8 @@ func (d *RootWalker) parsePHPDoc(n ir.Node, doc []phpdoc.CommentPart, actualPara
 			result.errs.pushType("%s on line %d", warning, part.Line())
 		}
 		param.typ = newTypesMap(&d.ctx, types)
-		param.typ.Iterate(func(t string) {
-			if t == "void" {
+		param.typ.Iterate(func(t meta.Type) {
+			if t.Is("void") {
```

Comment:
You may want to introduce `typ.Contains()` method or something like that instead?